README.md

```diff
@@ -95,7 +95,7 @@ where:
         withName: 'BUSCO' {  // Selects the process to apply settings.
             cpus     = 6     // Overrides cpu settings defined in nextflow.config
             time     = 4.d   // Overrides time settings defined in nextflow.config to 4 days. Use .h for hours, .m for minutes.
-            memory   = 20.G  // Overrides memory settings defined in nextflow.config to 20 GB.
+            memory   = '20G'  // Overrides memory settings defined in nextflow.config to 20 GB.
```

Comment:
It seem this should be GB actually:
https://www.nextflow.io/docs/latest/process.html#memory
```suggestion
            memory   = 20.GB // Overrides memory settings defined in nextflow.config to 20 GB.
```
```suggestion
            memory   = '20GB'  // Overrides memory settings defined in nextflow.config to 20 GB.
```

Reply:
OK. I only tested the original format `20.G` (gives me syntax error) and my proposed change `20G` (memory allocated correctly). But if it's in the manual I guess GB should work too.

Reply:
That '20G' works might be slurm specific. I'm surprised these aren't properly validated, but 🤷🏽 .
Select a commit and then I can merge the branch.